### DISCUSSION TITLE
Dirty shutdown of SO VM

### DISCUSSION THREAD

**Discussion by rangerrkm** [Security-Onion-Solutions/securityonion#3594] [Unsupported Versions]
Hello All,

Has anyone had this issue after a dirty shutdown/brownout. This is a Virtual Machine (Centos 7) SO version 2.3.21. 

![image](https://user-images.githubusercontent.com/25202205/112051460-95488a80-8b17-11eb-9122-d6773605c4fb.png)

We tried booting to a Linux Live USB and repairing the filesystem using xfs_repair, but with the root account locked we were not successful.

**Comment by weslambert** [comment]
You might try something similar to the following:
https://ask.fedoraproject.org/t/howto-cannot-open-access-to-console-the-root-account-is-locked-in-emergency-mode-dracut-emergency-shell/2010/2

**Comment by Acewiza** [comment]
Root issues aside, I just had a power failure followed by an onion that boots with most of the services failing to start, so I'll be watching this one until I either get time to troubleshoot it or just restore the last image...
